package.json

```diff
@@ -7,6 +7,7 @@
   "author": "Vladislav Kovalenko <kovalenkov222@gmail.com>, Kiril Yakymchuk <stbestich@gmail.com>",
   "license": "MIT",
   "scripts": {
+    "prebuild": "rimraf dist",
```

Comment:
What this line means?

Reply:
when you run 'build' script, first it will run 'prebuild' which deletes previous build, then run 'build'